tests/org.eclipse.fordiac.ide.test.ui/src/org/eclipse/fordiac/ide/test/ui/CreateFBTTest/CreateFBTTest.java

```diff
@@ -0,0 +1,236 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Prashantkumar Khatri
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Prashantkumar Khatri - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package org.eclipse.fordiac.ide.test.ui.CreateFBTTest;
+
+import static org.eclipse.swtbot.swt.finder.waits.Conditions.shellCloses;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+
+import org.eclipse.fordiac.ide.test.ui.Abstract4diacUITests;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Tree;
+import org.eclipse.swtbot.eclipse.finder.matchers.WidgetMatcherFactory;
+import org.eclipse.swtbot.eclipse.finder.widgets.SWTBotView;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotMenu;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotShell;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTree;
+import org.eclipse.swtbot.swt.finder.widgets.SWTBotTreeItem;
+import org.junit.jupiter.api.Test;
+
+public class CreateFBTTest extends Abstract4diacUITests {
+
+	/**
+	 * Checks if a New Type can be created and is visible in the System Explorer
+	 * With Empty Package Name
+	 *
+	 * The method checks if you can create a new Type via the menu, if the buttons
+	 * are enabled correctly, if you can assign a select a parent folder, enter type
+	 * name, packages and can select type template and at each stage check the state
+	 * of buttons and the UI responses. Then it checks if the newly created type can
+	 * be found in the hierarchy tree of the system explorer.
+	 *
+	 */
+	@SuppressWarnings("static-method")
+	@Test
+	public void createNewFBTypeWithoutPackage() {
+		final SWTBotMenu fileMenu = bot.menu(FILE).menu(NEW).menu(TYPE_PROJECT).click();
```

Comment:
can we use here the createFBType method that you itnroduced in Abstract4diacUITests?

Reply:
CreateFBType is for creating FB type where we need to, and this test method will test the behavior of UI components while creating a new FB type without a package name.